casr/Cargo.toml

```diff
@@ -52,5 +52,5 @@ name = "casr-dojo"
 required-features = ["dojo"]
 
 [dev-dependencies]
-lazy_static = "1.4"
+lazy_static = "1.5"
```

Comment:
Could we replace this crate with [LazyLock](https://doc.rust-lang.org/beta/std/sync/struct.LazyLock.html)?

Reply:
We agreed to do it in a separate PR